src/main/java/hudson/plugins/msbuild/MsBuildBuilder.java

```diff
@@ -320,6 +321,7 @@ static String[] tokenizeArgs(String args) {
     }
 
     @Extension @Symbol("msbuild")
+    @SuppressFBWarnings(value = "VO_VOLATILE_REFERENCE_TO_ARRAY", justification = "")
```

Comment:
Could you note in the justification that the reason this is volatile is not known?  An empty justification string doesn't help the next reader as much as being explicitly told, "I do not know why this was declared volatile".
```suggestion
    @SuppressFBWarnings(value = "VO_VOLATILE_REFERENCE_TO_ARRAY", justification = "untriaged")
```

Reply:
> Could you note in the justification that the reason this is volatile is not known? An empty justification string doesn't help the next reader as much as being explicitly told, "I do not know why this was declared volatile".

I will. I left it blank because I was not sure what to write in the justification. I figured it would be covered in the review. Maybe I should have a left comment myself.